lighthouse-core/audits/long-tasks.js

```diff
@@ -0,0 +1,125 @@
+/**
+ * @license Copyright 2020 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const Audit = require('./audit.js');
+const NetworkRequest = require('../lib/network-request.js');
+const NetworkRecords = require('../computed/network-records.js');
+const i18n = require('../lib/i18n/i18n.js');
+const MainThreadTasks = require('../computed/main-thread-tasks.js');
+
+const UIStrings = {
+  /** Title of a diagnostic LH audit that provides details on the longest running tasks that occur when the page loads. */
+  title: 'Avoids long-running tasks',
```

Comment:
Since we're just listing the tasks here and not scoring the audit, title should be more declarative than prescriptive: `Long tasks` should be fine.

description could be `Lists the longest tasks on the main thread`

Do we need any sort of web.dev url for this? maybe just link to web.dev/tbt ?

Reply:
I thought about linking to tbt, but unsure as I don't want this audit to be seen as specifically for tbt.